R/dash.R

```diff
@@ -176,16 +175,16 @@ Dash <- R6::R6Class(
       # ------------------------------------------------------------------------
       route <- routr::Route$new()
 
-      dash_index <- routes_pathname_prefix
+      dash_index <- self$config$routes_pathname_prefix
       route$add_handler("get", dash_index, function(request, response, keys, ...) {
```

Comment:
do we still need this given the catch all?

Reply:
Ah, good point. There's nothing like this handler in Dash for Python, so it definitely seems :hocho:-able.

Reply:
fixed in f0baaea64d062ee5b68013db83ef490ac3090b0b